eva-accession-clustering-automation/cluster_multiple_assemblies.py

```diff
@@ -34,8 +34,11 @@ def generate_bsub_command(assembly_accession, properties_path, clustering_artifa
     if memory:
         memory_amount = memory
 
-    command = 'bsub -J {job_name} -o {log_file} -e {error_file} -M {memory_amount} -R "rusage[mem={memory_amount}]" ' \
-              'java -jar {clustering_artifact} --spring.config.location=file:{properties_path}'\
+    command = 'bsub '
+    if dependency:
+        command += '-w {dependency} '.format(dependency=dependency)
```

Comment:
you are mixing `+=` concatenation with newline concatenation (`' ' \\n ' '`) and  `.format`ting. I suggest a more consistent approach.

Also, not sure if you know you can do this:
```suggestion
        command += '-w {} '.format(dependency)
```
which may or may not be clearer, depending on the particular code.

Reply:
@jmmut Please let me know if you think the new approach looks better for you